modules/mono/glue/GodotSharp/GodotSharp/Core/Aabb.cs

```diff
@@ -509,6 +509,62 @@ public readonly bool IntersectsPlane(Plane plane)
             return under && over;
         }
 
+        /// <summary>
+        /// Returns <see langword="true"/> if the <see cref="Aabb"/> intersects
+        /// the ray along <paramref name="dir"/> positioned at <paramref name="from"/>.
+        /// </summary>
+        /// <param name="origin">The origin of the ray.</param>
+        /// <param name="dir">The direction of the ray.</param>
+        /// <returns>
+        /// A <see langword="bool"/> for whether or not the <see cref="Aabb"/> intersects the ray.
+        /// </returns>
+        public readonly bool IntersectsRay(Vector3 from, Vector3 dir)
+        {
+            if (HasPoint(from)) return true;
+
```

Comment:
Instead of using the `End` property, we usually match the Core implementation and create an aux variable like this:

```suggestion

            Vector3 end = _position + _size;
```